app-text/pdf2html/pdf2html-1.4.ebuild

```diff
@@ -12,13 +12,14 @@ SRC_URI="ftp://atrey.karlin.mff.cuni.cz/pub/local/clock/pdf2html/${P}.tgz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 ~ppc sparc x86 ~amd64-linux ~x86-linux ~ppc-macos"
-IUSE=""
 
 DEPEND=">=media-libs/libpng-1.4
-	sys-libs/zlib"
+	sys-libs/zlib
+"
 RDEPEND="${DEPEND}
 	app-text/ghostscript-gpl
-	>=media-gfx/imagemagick-6"
+	>=media-gfx/imagemagick-6
+"
```

Comment:
These are usually maintainer-chosen styling preferences - you shouldn't touch this.